src/ape/managers/chain.py

```diff
@@ -0,0 +1,360 @@
+import time
+from typing import Dict, Iterator, List, Optional, Union
+
+from dataclassy import dataclass
+
+from ape.api import AddressAPI, BlockAPI, ProviderAPI, ReceiptAPI, TestProviderAPI
+from ape.exceptions import ChainError, ProviderNotConnectedError, UnknownSnapshotError
+from ape.logging import logger
+from ape.types import AddressType, BlockID, SnapshotID
+
+from .networks import NetworkManager
+
+
+@dataclass
+class BlockContainer:
+    """
+    A list of blocks on the chain.
+
+    Usages example::
+
+        from ape import chain
+
+        latest_block = chain.blocks[-1]
+
+    """
+
+    networks: NetworkManager
+    _poll_wait_interval = 2  # Seconds
```

Comment:
Config?

Reply:
how about we just make this is a param on the `poll` method and let it default to 2?